.github/workflows/rust.yml

```diff
@@ -0,0 +1,55 @@
+name: Rust
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [1.77.2]
```

Comment:
I don't think we need support to test multiple rust versions, do we? What about moving that up to `env`?

Reply:
Good point. I removed it. I believe that we don't even need it because we have the rust-toolchain file